lib/dialects/abstract/index.js

```diff
@@ -39,7 +39,8 @@ AbstractDialect.prototype.supports = {
   inserts: {
     ignoreDuplicates: '', /* dialect specific words for INSERT IGNORE or DO NOTHING */
     updateOnDuplicate: false, /* whether dialect supports ON DUPLICATE KEY UPDATE */
-    onConflictDoNothing: '' /* dialect specific words for ON CONFLICT DO NOTHING */
+    onConflictDoNothing: '', /* dialect specific words for ON CONFLICT DO NOTHING */
+    conflictFields: false /* whether the dialect supports specifying conflict fields or not */
```

Comment:
Is this not supported for mysql and mssql?

Reply:
Nope; same for snowflake iirc.  From what I gathered whilst looking at their respective docs, they don't let / make you explicitly specify the fields which could run into a `UNIQUE` constraint issues.  Instead, they have an `ON DUPLICATE` clause which is triggered  when *any* unique constraint would error.